_targets.R

```diff
@@ -7,9 +7,35 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
+  tar_target(
+    site_data_01427207,
+    download_nwis_site_data("01427207"),
```

Comment:
We also typically include the filepath for the out_file, so that anyone reading the `_targets.R` file can see where these file targets are saved